upup/pkg/fi/cloudup/awsup/mock_aws_cloud.go

```diff
@@ -308,7 +308,7 @@ func (c *MockAWSCloud) DescribeInstanceType(instanceType string) (*ec2.InstanceT
 	}
 
 	switch instanceType {
-	case "c5.large", "m3.medium", "m4.large", "m5.large", "m5.xlarge", "t2.micro", "t2.medium", "t3.medium", "t3.large":
+	case "c5.large", "m3.medium", "m4.large", "m5.large", "m5.xlarge", "t2.micro", "t2.medium", "t3.medium", "t3.large", "c4.large":
```

Comment:
Not sure what changes for "c4.large" to be added here.

Reply:
Just for tests to pass.. :)

Reply:
I don't think anything there will be any sort of issues with this..

Reply:
👍